idl/replication.thrift

```diff
@@ -31,3 +31,10 @@ struct query_cfg_response
     4:bool is_stateful;
     5:list<partition_configuration> partitions;
 }
+
+struct request_meta {
```

Comment:
I don`t suggest to use optional in java client. We use optional in pegasus server, because we want it to have good compatibility, so we can delete some data in later java client versions. But in this version of java client, these data is a must.